src/dbt/kipptaf/models/extracts/tableau/rpt_tableau__college_assessment_dashboard.sql

```diff
@@ -87,14 +119,32 @@ with
             end as subject_area,
             score as scale_score,
             rn_highest,
+            if(
+                cast(format_date('%m', date) as int) <= 7,
+                cast(format_date('%Y', date) as int) - 1,
+                cast(format_date('%Y', date) as int)
+            ) as test_academic_year,
+            case
+                when
+                    score_type in (
+                        'act_reading',
+                        'act_english',
+                        'sat_reading_test_score',
+                        'sat_ebrw'
+                    )
+                then 'ENG'
+                when score_type in ('act_math', 'sat_math_test_score', 'sat_math')
+                then 'MATH'
+                else 'NA'
+            end as course_discipline,

```

Comment:
would these be useful to add to `int_kippadb__standardized_test_unpivot`?

Reply:
course_discipline would be super useful to add. I didn't use SCI here, but that one would also be helpful! I'll add them and await your approval.

Reply:
actually, this is a bit more complex than I think we'd want on this query. i added an agenda item to our O5 today.